src/backend/utils/misc/guc.c

```diff
@@ -8200,7 +8200,7 @@ static struct config_string ConfigureNamesString[] =
 			NULL
 		},
 		&standby_addr_host,
-		"localhost", NULL, NULL
+		"none", NULL, NULL
```

Comment:
So that's why we need another fix if necessary. the fix should fix script and code together. @paul-guo-